.github/workflows/kind-e2e.yaml

```diff
@@ -31,22 +31,22 @@ jobs:
         include:
           # Map between K8s and KinD versions.
           # This is attempting to make it a bit clearer what's being tested.
-          # See: https://github.com/kubernetes-sigs/kind/releases/tag/v0.9.0
-        - k8s-version: v1.18.15
-          kind-version: v0.10.0
-          kind-image-sha: sha256:5c1b980c4d0e0e8e7eb9f36f7df525d079a96169c8a8f20d8bd108c0d0889cc4
-          kingress: kourier
-          cluster-suffix: c${{ github.run_id }}.local
-        - k8s-version: v1.19.7
-          kind-version: v0.10.0
-          kind-image-sha: sha256:a70639454e97a4b733f9d9b67e12c01f6b0297449d5b9cbbef87473458e26dca
+          # See: https://github.com/kubernetes-sigs/kind/releases
+        - k8s-version: v1.19.11
+          kind-version: v0.11.0
```

Comment:
what about 0.11.1, @nak3 ?

Reply:
Sure!  Updated.
0.11.1 fixed runc's vulnerability and it should not effect on our CI but the latest version is better anyway :smile: